apps/src/music/player/MusicPlayer.ts

```diff
@@ -180,7 +184,7 @@ export default class MusicPlayer {
       blockId: 'preview',
       soundType: 'beat',
     };
-
+    this.analyticsReporter?.onSoundsPlayed(id);
```

Comment:
The result of `this.convertEventSamples(preview)[0]` that's computed below should return a sample that has a URL, in the same format as the sample that's passed to `scheduleSample` in ToneJSPlayer. So you should be able to use the same function to convert the URL back to an ID. It's a little roundabout, but will ensure that we're using the same ID format consistently for reporting.

Reply:
Thanks! will update.